src/languages/lisp.md

```diff
@@ -0,0 +1,168 @@
+
+- Your `.asd` file is named like your system name. E.g. `example.asd` will have `(defsystem example ...)`.
+
+Platform.sh will then run `(asdf:make :example)` on your system to build a binary.
+
+If you don't want these assumptions, you can disable this behavior by specifying in your `.platform.app.yaml`:
+
+```yaml
+build:
+    flavor: none
+```
+
+## Dependencies
+
+The recommended way to handle Lisp dependencies on Platform.sh is using ASDF. Commit a `.asd` file in your repository and the system will automatically download the dependencies using QuickLisp.
+
+## QuickLisp options
+
+If you wish to change the distributions that QuickLisp is using, you can specify those as follows, specifying a distribution name, its URL and, an optional version:
+
+```yaml
+runtime:
+    quicklisp:
+        <distribution name>:
+            url: "..."
+            version: "..."
+```
+
+For example:
+
+```yaml
+runtime:
+    quicklisp:
+        quicklisp:
+            url: 'http://beta.quicklisp.org/dist/quicklisp.txt'
+            version: '2019-07-11'
+```
+
+
+## Platform.sh variables
+
+Platform.sh exposes relationships and other configuration as [environment variables](/development/variables.md). Most notably, it allows a program to determine at runtime what HTTP port it should listen on and what the credentials are to access [other services](/configuration/services.md).
+
+To get the `PORT` environment variable (the port on which your web application is supposed to listen) you would:
+
+```lisp
+(parse-integer (uiop:getenv "PORT"))
+```
+
+
+## Building and running the application
+
+Assuming `example.lisp` and `example.asd` are present in your repository, the application will be automatically built on push.  You can then start it from the `web.commands.start` directive.  Note that the start command _must_ run in the foreground. Should the program terminate for any reason it will be automatically restarted. In the example below we sleep for a very, very long time. You could also choose to join the thread of your web server, or use other methods to make sure the program does not terminate.
+
+The following basic `.platform.app.yaml` file is sufficient to run most Lisp applications.
+
+```yaml
+name: app
+type: lisp:1.5
+web:
+    commands:
+        start: ./example
+    locations:
+        /:
+            allow: false
+            passthru: true
+disk: 512
+```
+
+Note that there will still be a proxy server in front of your application.  If desired, certain paths may be served directly by our router without hitting your application (for static files, primarily) or you may route all requests to the Lisp application unconditionally, as in the example above.
+
+# Accessing Services
+
+The services configuration is available in the environment variable `PLATFORM_RELATIONSHIPS`. 
+
+To parse them, add the dependencies to your `.asd` file:
+
+```lisp
+:depends-on (:jsown :babel :s-base64)
+````
```

Comment:
```suggestion
```
```

Reply:
Alright, tried to just suggest a change on these two points, but suggestions use Markdown ticks too, so it doesn't show up correctly. 

So don't commit my suggestion, since it looks like it will remove everything if you do.

For here and below, change the four ticks ("````") to three ticks ("```") so that they match the number at the beginning of the code block. Otherwise, the fourth shows up in docs on an extra line (https://pr-1171-gs7ujbi-652soceglkw4u.eu-3.platformsh.site/languages/lisp.html#accessing-services)